tutorials/developer-tutorials/createNewIdentityAttributes.md

```diff
@@ -0,0 +1,168 @@
+
+***Save*** the changes.
+
+The above simply checks if the identity trying to access the `createCar()` function has the attribute `manufacturer` set to `true`. If it does, allow the transaction to be executed. If the identity does not have the attribute `manufacturer` (or it isn't set to true), do not let the identity carry out the transaction and show an error message.
+
+> Alternatively, you could use the <a href='https://fabric-shim.github.io/release-1.4/fabric-contract-api.Contract.html#beforeTransaction__anchor'> beforeTransaction()</a> method. This method could be used to seperate your business logic from your permissions.
+
+3. To ensure that the above changes are included within our contract, we must `package`, `install` and then `instantiate` the contract.
+
+<details>
+<summary><b>Package smart contract</b></summary>
+
+1. In the left sidebar, click on the __IBM Blockchain Platform__ icon.
+
+2. Mouse-over the `SMART CONTRACTS` panel, click the `...` menu, and select `Package Open Project` from the dropdown.
+ 
+> Command Palette alternative: `Package Open Project`
+
+3. You should see a new package on the list, `carContract@0.0.1`, if everything went well.
+
+</details>
+
+<details>
+<summary><b>Install the smart contract</b></summary>
+
+To do this...
+
+1. In the `Fabric Environments` panel, look for `+ Install` (it's under Smart Contracts > Installed) and click it.
+
+2. You'll be asked to choose a package to install. Pick `carContract@0.0.1`.
+
+You should see `carContract@0.0.1` appear under the Smart Contracts > Installed list.
+
+> Command Palette alternative: `Install Smart Contract`
+
+That's it - job done! Next up, we'll instantiate the smart contract...
+
+</details>
+
+
+<details>
+<summary><b>Instantiate the smart contract</b></summary>
+
+To instantiate our installed contract...
+
+1. In the `Fabric Environments` panel, look for `+ Instantiate` (it's under Smart Contracts > Instantiated) and click it.
+
+2. You'll be asked to choose a smart contract to instantiate. Pick `carContract@0.0.1`.
+
+3. You'll be asked what function to call. If you wanted to use a specific function as part of your instantiate, you could enter something here.  We'll see that happen in future tutorials, but for now just hit `Enter` to skip this step.
+
+4. You'll be asked if you want to provide a private data configuration file. For this tutorial just click `No`, in future tutorials we will explain more about this.
+
+Instantiation will take a while longer than install - watch out for the success message and `carContract@0.0.1` appearing in the Smart Contracts > Instantiated list to confirm it's worked!
+
+> Command Palette alternative: `Instantiate Smart Contract`
+</details>
+
+4. To test that attribute-based access control works, we can connect to the gateway by pressing `Local Fabric` under the `FABRIC GATEWAYS` panel. You will now be asked which identity you wish to connect with - choose `Jack` (created in our previous tutorial). 
```

Comment:
I think this should be a new section of showing what happens when an attribute causes an error

Reply:
Fixed